ci/integration-tests-windows.groovy

```diff
@@ -76,9 +89,9 @@ node('py36') {
                                 rm -f ./env/Scripts/dcos.exe; \
                                 python -c 'import urllib.request; urllib.request.urlretrieve(\\\"https://downloads.dcos.io/cli/testing/binaries/dcos/windows/x86-64/master/dcos.exe\\\", \\\"dist/dcos.exe\\\")'; \
                                 PATH=$PWD/dist:$PATH; \
-                                dcos cluster remove --all; \
-                                dcos cluster setup ${DCOS_TEST_URL} --insecure; \
-                                dcos plugin add ../../../build/windows/dcos-core-cli.zip; \
+                                dcos -vv cluster remove --all; \
+                                dcos -vv cluster setup ${DCOS_TEST_URL} --insecure; \
+                                dcos -vv plugin add ../../../build/windows/dcos-core-cli.zip; \
```

Comment:
Not sure if we want fully verbose logging, I'm not sure if that might reveal secrets or not. At least `-v` would be helpful though.

Reply:
Ok will wait for @bamarni to take a last look and update the PR accordingly.

Reply:
Even though Jenkins would hide credentials, safer to remove the verbose flag to leave it as before.